bi_sql_editor/tests/test_bi_sql_view.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase, at_install, post_install
+from odoo.exceptions import AccessError
+
+
+class TestBiSqlViewEditor(TransactionCase):
+
+    def setUp(self):
+        super(TestBiSqlViewEditor, self).setUp()
+        self.res_partner = self.env['res.partner']
+        self.res_users = self.env['res.users']
+        self.bi_sql_view = self.env['bi.sql.view']
+        self.group_bi_user = self.env.ref(
+            'sql_request_abstract.group_sql_request_manager')
+        self.group_user = self.env.ref(
+            'base.group_user')
+        self.view = self.bi_sql_view.create({
+            'name': 'Partners View 2',
+            'is_materialized': False,
+            'technical_name': 'partners_view_2',
+            'query': "SELECT name as x_name, street as x_street,"
+                     "company_id as x_company_id FROM res_partner "
+                     "ORDER BY name"
+        })
+        self.company = self.env.ref('base.main_company')
+        # Create bi user
+        self.bi_user = self._create_user('bi_user', [self.group_bi_user],
+                                         self.company)
+        self.no_bi_user = self._create_user('no_bi_user', [self.group_user],
+                                            self.company)
+
+    def _create_user(self, login, groups, company):
+        """Create a user."""
+        group_ids = [group.id for group in groups]
+        user = self.res_users.create({
+            'name': 'Test BI User',
+            'login': login,
```

Comment:
Test are good. Just are this here to make it not fail:
`'notify_email': 'none',`

Running the test multiple times won't create multiple bi_sql_view records, that's good. :smile: